src/tools/schema2graph.ts

```diff
@@ -92,6 +100,12 @@ export class SchemaNode {
     return humanNames.sort()[0];
   }
 
+  static singleSchemaFullName(connection: HandleConnectionSpec, nodes: SchemaNode[]): string {
```

Comment:
Is singleSchemaHumanName still used? If not, please remove.

Reply:
It is, twice in schema2kotlin